docs/en/fuse-sdk/FUSE-SDK-Dora-Quick-Start.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: global
+title: FUSE SDK with Distributed Cache Quick Start
+group: FUSE SDK
+priority: 3
+---
+
+* Table of Contents
+  {:toc}
+
+![POSIX LOCAL CACHE](https://github.com/Alluxio/alluxio/blob/dora/docs/img/posix-distributed-cache.png)
```

Comment:
move images into resources so they get uploaded with the docs and served the same way instead of referencing a github url

Reply:
Changed, this is to let users see the docs previously without needing to use Alluxio documentation page